pkg/services/ngalert/notifier/compat.go

```diff
@@ -46,47 +46,51 @@ func PostableApiAlertingConfigToApiReceivers(c apimodels.PostableApiAlertingConf
 
 type DecryptFn = func(value string) string
 
-func PostableToGettableGrafanaReceiver(r *apimodels.PostableGrafanaReceiver, provenance *models.Provenance, decryptFn DecryptFn) (apimodels.GettableGrafanaReceiver, error) {
+func PostableToGettableGrafanaReceiver(r *apimodels.PostableGrafanaReceiver, provenance *models.Provenance, decryptFn DecryptFn, listOnly bool) (apimodels.GettableGrafanaReceiver, error) {
```

Comment:
It would be great if could keep this compat method clean and remove fields either before this func or later in the service. That's not a blocker, though.

Reply:
I was 50:50 on that, let's definitely discuss for a future PR.